test/extensions/soulbound.ts

```diff
@@ -106,51 +110,122 @@ describe('RMRKSoulboundNestableExternalEquippableMock', async function () {
   shouldBehaveLikeSoulboundNestable();
 });
 
-describe('RMRKSoulbound exempt', async function () {
-  let token: RMRKSemiSoulboundNestableMock;
+describe('RMRKSoulbound variants', async function () {
   let owner: SignerWithAddress;
   let otherOwner: SignerWithAddress;
 
   beforeEach(async function () {
-    const signers = await ethers.getSigners();
-    owner = signers[0];
-    otherOwner = signers[1];
-    const factory = await ethers.getContractFactory('RMRKSemiSoulboundNestableMock');
-    token = <RMRKSemiSoulboundNestableMock>await factory.deploy('Chunky', 'CHNK');
-    await token.deployed();
+    [owner, otherOwner] = await ethers.getSigners();
   });
 
-  it('can support IRMRKSoulbound', async function () {
-    expect(await token.supportsInterface(IRMRKSoulbound)).to.equal(true);
+  describe('RMRKSoulbound After Blocks', async function () {
```

Comment:
Should this equal the contact name?

```suggestion
  describe('RMRKSoulboundAfterBlockNumber', async function () {
```